lib/compat/wordpress-6.6/admin-bar.php

```diff
@@ -32,7 +32,7 @@ function gutenberg_admin_bar_edit_site_menu( $wp_admin_bar ) {
 	$wp_admin_bar->add_node(
 		array(
 			'id'    => 'site-editor',
-			'title' => __( 'Site Editor' ),
+			'title' => __( 'Edit site' ),
```

Comment:
I'm guessing we need to use 'Manage site' for multi-site here? Can we add a similar check to the one used in Core?

Reply:
or maybe this one only overrides the single site menu?

Reply:
I haven't check for multisite, will check and let you know.

Reply:
@youknowriad adding this will work for multisite right?
```php
if ( is_blog_admin() && is_multisite() && current_user_can( 'manage_sites' ) ) {
		$wp_admin_bar->add_node(
			array(
				'parent' => 'site-name',
				'id'     => 'edit-site',
				'title'  => __( 'Manage Site' ),
				'href'   => network_admin_url( 'site-info.php?id=' . get_current_blog_id() ),
			)
		);
	}
```

Reply:
This could work but reading again the purpose of the override of the code in Gutenberg, We actually just want to change the "url" for single site, so maybe a simple solution here is to just return early when we're in multisite (rely on core)

Reply:
@youknowriad so I suggest we can change this early return to below.
```php
// Before
if ( ! current_user_can( 'edit_theme_options' ) || is_admin() ) {
		return;
	}

// After
if ( ! current_user_can( 'edit_theme_options' ) || is_admin() || ( is_blog_admin() && is_multisite() && current_user_can( 'manage_sites' ) ) ) {
		return;
	}
```

Reply:
Works for me.